### ISSUE TITLE
Command/navigation handling error

### ISSUE THREAD

**Issue by ArrayParadigm** [ArrayParadigm/Shinobi_mud#2]
```
Login successful!
You are in Room 3000. This room was auto-added.
Exits: 
You are alone in this room.
look
Room 3000: A serene entrance with soft ambient light, welcoming you to Eve's Hav
en.
Exits: north, east
north
Moved to room 3001.
You are in Room 3001. This room was auto-added.
Exits: 
You are alone in this room.
south
Moved to room 3000.
You are in Room 3000. This room was auto-added.
Exits: 
You are alone in this room.
l
Unknown command.
loo
Unknown command.
look
Room 3000: A serene entrance with soft ambient light, welcoming you to Eve's Hav
en.
Exits: north, east
north
Moved to room 3001.
You are in Room 3001. This room was auto-added.
Exits: 
You are alone in this room.
look
Room 3001: A tranquil garden, filled with luminous flora swaying gently.
```

Room loading and descriptions are not loading correctly. it's attempting to process the admin/builder side before the mud usage side. as a result, it'll show that the room is being made or something until using a command that's processed by general_commands.   

TLDR; need to work with the order to switch the stack so that general navigation/room validation runs before admin_validation
